src/search/heuristics/cea_heuristic.cc

```diff
@@ -412,6 +413,12 @@ ContextEnhancedAdditiveHeuristic::ContextEnhancedAdditiveHeuristic(
     const plugins::Options &opts)
     : Heuristic(opts),
       min_action_cost(task_properties::get_min_operator_cost(task_proxy)) {
+
+    if (task_properties::has_axioms(task_proxy)) {
+        task = make_shared<tasks::NegatedAxiomsTask>(tasks::NegatedAxiomsTask(task));
+        task_proxy = TaskProxy(*task);
+    }
```

Comment:
I locally tried Christian's tasks with the cea heuristic (so a setting where we only do the transformation once) and that got stuck for at least 10 minutes, so these tasks are already unmanageable when only computing the transformation once. But yes, it would be even worse with lama-first :).